### PR TITLE
mobile: Enable CI  for C++ tests with xDS and full protos enabled

### PR DESCRIPTION
**Pull request by fredyw**
Risk Level: low
Testing: CI
Docs Changes: n/a
Release Notes: n/a
Platform Specific Features: n/a

### CODE REVIEW

File: .github/workflows/mobile-compile_time_options.yml [hunk 1]
```diff
@@ -56,6 +56,18 @@ jobs:
             build
             --config=mobile-remote-ci-cc-no-exceptions
             //test/performance:test_binary_size //library/cc/...
+        - name: Running C++ tests with xDS enabled
+          target: cc-tests-xds-enabled
+          args: >-
+            test
+            --config=mobile-remote-ci-cc-xds-enabled
+            //test/common/... //test/cc/...
```
Comment (abeyad) [inline .github/workflows/mobile-compile_time_options.yml:64]: can only the xDS tests be run for this?

Comment (fredyw) [reply .github/workflows/mobile-compile_time_options.yml]: Done.

**Review by abeyad — APPROVED** [review]

**Pull request merged**